src/model/account/RestrictionType.ts

```diff
@@ -15,15 +15,15 @@
  */
 
 /**
- * Account property type
- * 0x01	The property type is an address.
- * 0x02	The property type is mosaic id.
- * 0x03	The property type is a transaction type.
- * 0x04	Property type sentinel.
- * 0x80 + type	The property is interpreted as a blocking operation.
+ * Account restriction type
+ * 0x01	The restriction type is an address.
+ * 0x02	The restriction type is mosaic id.
+ * 0x03	The restriction type is a transaction type.
+ * 0x04	restriction type sentinel.
+ * 0x80 + type	The restriction is interpreted as a blocking operation.
  */
 
-export enum PropertyType {
+export enum RestrictionType {
     AllowAddress = 0x01,
     AllowMosaic = 0x02,
     AllowTransaction = 0x04,
```

Comment:
consistency problem: in doccomment `0x04` is "restriction type sentinel" while here it represents a transaction type.